deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -94,6 +110,8 @@ export class StackComposer {
         if (!contextJSON) {
             throw new Error(`No CDK context block found for contextId '${contextId}'`)
         }
+        console.log('Received following context block for deployment: ')
+        console.log(contextJSON)
```

Comment:
thanks! I'd recommend putting a trailer after though so that it's clear where it ends.

Reply:
Added 👍